pkg/cmd/server/apis/config/v1/types.go

```diff
@@ -228,9 +228,6 @@ type MasterConfig struct {
 	// ControllerConfig holds configuration values for controllers
 	ControllerConfig ControllerConfig `json:"controllerConfig"`
 
-	// DisabledFeatures is a list of features that should not be started.
-	DisabledFeatures FeatureList `json:"disabledFeatures"`
```

Comment:
can we give users a deprecation warning here and ignore that option? Might help migration if somebody used this before.

Reply:
It never even worked, so I don't think it makes any sense. I wonder if we have any docs about it at all.